tests/src/main/python/rest/tests/extract/.travis.yml

```diff
@@ -0,0 +1,13 @@
+# ref: https://docs.travis-ci.com/user/languages/python
```

Comment:
I don't think we actually use travis?
Is there a away to turn off the autogeneration of this file to avoid confusion?